verify.py.in

```diff
@@ -12,12 +12,31 @@ import re
 import sys
 
 SEAHORN_ROOT = "@SEAHORN_ROOT@"
-#ERROR_PREFIX = r'^Error:.*(Antecedent|Consequent).*$'
-ERROR_PREFIX = r'^Error: (vacuity|assertion) failed'
+ASSERT_ERROR_PREFIX = r'^Error: assertion failed'
+# the plan is to have two sets, vac error and info and put filepath:linenumbers) into both
+VACUITY_CHECK_RE = r'^(?P<stream>Info|Error).*(?P<what>vacuity).*(?P<result>passed|failed).*sat\) (?P<debuginfo>.*)$'
+
+def check_vacuity(line, passed_set, failed_set):
+    m = re.match(VACUITY_CHECK_RE, line)
+    if not m:
+        return
+    debugInfo = m.group('debuginfo')
+    if (m.group('stream') == 'Info'
+        and m.group('what') == 'vacuity'
+        and m.group('result') == 'passed'):
+        passed_set.add(debugInfo.strip())
+    elif (m.group('stream') == 'Error'
+        and m.group('what') == 'vacuity'
+        and m.group('result') == 'failed'):
+        failed_set.add(debugInfo.strip())
+
 
 def main(argv):
     import sea
 
+    def check_vacuity_inner(line, passed_set, failed_set):
```

Comment:
why is `check_vacuity_inner` needed? The code can just call `check_vacuity`.

The name `check_vacuity` is misleading since no check is happening, I would have called it `process_vacuity_stat` or something like that